TrafficCapture/trafficCaptureProxyServerTest/src/main/docker/docker-compose.yml

```diff
@@ -1,6 +1,30 @@
 version: '3.7'
 services:
   webserver:
-    image: 'nginx-simple:latest'
+    image: 'migrations/nginx-perf-test-webserver:latest'
     ports:
-      - "80:80"
+      - "8080:80"
+
+  captureproxy:
+    image: 'migrations/capture_proxy:latest'
+    networks:
+      - testing
+    ports:
+      - "9201:9201"
+    command: /runJavaWithClasspath.sh org.opensearch.migrations.trafficcapture.proxyserver.Main --destinationUri  http://webserver:80 --listenPort 9201 --noCapture
+    depends_on:
+      - webserver
+
+  jmeter:
+    image: 'migrations/jmeter:latest'
+    networks:
+      - testing
+    command: /bin/sh -c "while sleep 10; do /runJavaWithClasspath.sh org.opensearch.migrations.trafficcapture.JMeterLoadTest -p 9201 -s captureproxy; done"
```

Comment:
Please kill the while part.  If you want, you can put a tail -f /dev/null in at the end to keep the container alive to make it easier to run additional commands or process files.  I'd like to know when a run has been done and have whatever time I need to process the results across the other containers (manually or automatically).